benchmarks/math/triangulate.gd

```diff
@@ -0,0 +1,26 @@
+extends Benchmark
+
+const RADIUS := 1.0
+const NUM_SIDES := 1_000
+const POSITION := Vector2(0.0, 0.0)
+
+func generate_circle_polygon(radius: float,
+							 num_sides: int,
+							 position: Vector2) -> PackedVector2Array:
+	var angle_delta: float = (PI * 2) / num_sides
+	var vector: Vector2 = Vector2(radius, 0)
+	var polygon := PackedVector2Array()
+
+	for _i in num_sides:
+		polygon.append(vector + position)
+		vector = vector.rotated(angle_delta)
+
+	return polygon
+
+var circle := generate_circle_polygon(RADIUS, NUM_SIDES, POSITION)
+
+func _init() -> void:
+	test_idle = true
```

Comment:
This part is ineffective because idle time reporting only works for long-running benchmarks. You don't need it for what you're doing here anyway – relying on the Main Thread Time (which is always enabled) is enough, so you can remove that line.